CONTRIBUTING.md

```diff
@@ -95,8 +136,29 @@ After the approval the pull request can be merged into the repository. This is d
 
 ## Post-Merge Cleanup
 **Congratulation.**
-You successfully contributed to the aas-spec repository.
+You successfully contributed to a specification repository of the Asset Administration Shell.
 
 If you are a member of the development team, please delete the feature branch you directly created within the aas-specs repository.
 
 Otherwise, if you are not part of the team and you forked the repository, feel free to delete your fork.
+
+
+## License Headers & Licensing
+
+All files contributed require headers - this will ensure the license and copyright clearing at the end. Also, all contributions must have the same license as the source.
```

Comment:
JSON doesn't allow headers. Therefore, maybe write it like this:
"All files contributed require headers ..." --> "By default, all files contributed require headers  - this will ensure the license and copyright clearing at the end. Only if inline comments are not possible, e.g., for JSON files, the copyright declaration can be omitted. ..."